src/components/NewModal/index.styles.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import { spacing, colors, constants } from "../../theme/index";
+import Row from "../Grid/Row";
+
+export const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
```

Comment:
I think we need to introduce style variants for this component since the `Modal` will not always be full-screen.